[2022-10-24 12:01:14,571] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_training_pipeline.training_model manual__2022-10-24T12:00:51.724366+00:00 [queued]>
[2022-10-24 12:01:14,576] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_scoring_training_pipeline.training_model manual__2022-10-24T12:00:51.724366+00:00 [queued]>
[2022-10-24 12:01:14,576] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-10-24 12:01:14,576] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-10-24 12:01:14,576] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-10-24 12:01:14,584] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): training_model> on 2022-10-24 12:00:51.724366+00:00
[2022-10-24 12:01:14,591] {standard_task_runner.py:52} INFO - Started process 3809 to run task
[2022-10-24 12:01:14,597] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_scoring_training_pipeline', 'training_model', 'manual__2022-10-24T12:00:51.724366+00:00', '--job-id', '258', '--raw', '--subdir', 'DAGS_FOLDER/Lead_scoring_training_pipeline/.ipynb_checkpoints/lead_scoring_training_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmpyw5hlx4w', '--error-file', '/tmp/tmp5ngswqmz']
[2022-10-24 12:01:14,597] {standard_task_runner.py:80} INFO - Job 258: Subtask training_model
[2022-10-24 12:01:14,648] {task_command.py:371} INFO - Running <TaskInstance: Lead_scoring_training_pipeline.training_model manual__2022-10-24T12:00:51.724366+00:00 [running]> on host 2d7632d022aa
[2022-10-24 12:01:14,697] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_scoring_training_pipeline
AIRFLOW_CTX_TASK_ID=training_model
AIRFLOW_CTX_EXECUTION_DATE=2022-10-24T12:00:51.724366+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-24T12:00:51.724366+00:00
[2022-10-24 12:01:16,347] {logging_mixin.py:115} WARNING - /opt/conda/lib/python3.8/site-packages/sklearn/utils/validation.py:72 DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
[2022-10-24 12:01:24,939] {logging_mixin.py:115} WARNING - Registered model 'LightGBM' already exists. Creating a new version of this model...
[2022-10-24 12:01:24,964] {logging_mixin.py:115} WARNING - 2022/10/24 12:01:24 INFO mlflow.tracking._model_registry.client: Waiting up to 300 seconds for model version to finish creation.                     Model name: LightGBM, version 9
[2022-10-24 12:01:24,965] {logging_mixin.py:115} WARNING - Created version '9' of model 'LightGBM'.
[2022-10-24 12:01:25,411] {logging_mixin.py:115} INFO - get_trained_model has been executed successfully.
[2022-10-24 12:01:25,424] {python.py:173} INFO - Done. Returned value was: None
[2022-10-24 12:01:25,439] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_scoring_training_pipeline, task_id=training_model, execution_date=20221024T120051, start_date=20221024T120114, end_date=20221024T120125
[2022-10-24 12:01:25,472] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-24 12:01:25,490] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
